### PR TITLE
Added local attraction to cape coral readme

### PR DESCRIPTION
**Pull request by JakeRuss**
This pull request is to add Cape Coral to cities

### CODE REVIEW

**Comment by setgetme** [comment]
@JakeRuss, What type of park is it?

File: jakeruss-capecoral.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Cape Coral, FL
+
+### Restaurants
+
+### Tourist Attractions
+
+### Local Attractions
+- Cayo Costa State Park
```
Comment (githubteacher) [inline jakeruss-capecoral.md]: @JakeRuss is this an all-day park? Is there enough stuff to do to take up your entire day?

Comment (JakeRuss) [inline jakeruss-capecoral.md]: It is an island in the gulf, take the ferry over to the beach, or bring camping gear!

**Comment by JakeRuss** [comment]
It's an island with beaches and camping grounds

**Pull request merged**